packages/shared/src/components/LoginButton.tsx

```diff
@@ -33,15 +33,15 @@ const getLogEvent = (copy: ButtonCopy): LogEvent => ({
 export default function LoginButton({
   className = {},
 }: LoginButtonProps): ReactElement {
-  const { showLogin } = useContext(AuthContext);
+  const router = useRouter();
   const { logEvent } = useContext(LogContext);
 
   const onClick = (copy: ButtonCopy) => {
     logEvent(getLogEvent(copy));
-    showLogin({
-      trigger: AuthTriggers.MainButton,
-      options: { isLogin: copy === ButtonCopy.Login },
-    });
+    const params = new URLSearchParams();
+    params.set('authTrigger', AuthTriggers.MainButton);
+    params.set('afterAuth', window.location.pathname);
+    router.push(`/onboarding?${params.toString()}`);
```

Comment:
There might actually be another edge-case we need to support if they come from a existing campaign we need to forward existing search params. (marketing doodles and userId etc)